heartbeat/heartbeat.reference.yml

```diff
@@ -465,7 +465,7 @@ output.elasticsearch:
   # if no error is encountered.
   #slow_start: false
 
-  # Optional index name. The default index name is set to name of the beat
+  # Optional index name. The default index name is set to the index-prefix of the beat
```

Comment:
I'm think what we should name this? Probably best would be `beat-index-prefix` as before it was generic but having the the `beat-index-prefix` replacement inside should be valid.